app/assets/stylesheets/public/_nav.scss

```diff
@@ -127,6 +127,7 @@
 
     &--level2 {
       @media (min-width: $screen-md) {
+        background-color: #fff;
```

Comment:
In case of content overlap, the white background will cover up what's behind the sidebar